app/src/main/java/com/example/bigowlapp/activity/SendSmsInvitationActivity.java

```diff
@@ -18,18 +19,38 @@
 public class SendSmsInvitationActivity extends AppCompatActivity {
 
     private EditText number, message;
-    private Button send, testSendRequest;
-    private String number2, message2;
+    private Button send;
+    private String number2, message2, noteText;
```

Comment:
I think you should put a more descriptive name for the variable. The name itself should explain what it does. If someone has to refactor the code or debug, they should not spend too much time figuring out what the variable does.

Reply:
ok. I have renamed the variables